std/algorithm/iteration.d

```diff
@@ -425,6 +431,13 @@ returns a range of which elements are obtained by applying $(D fun(a))
 left to right for all elements $(D a) in $(D range). The original ranges are
 not changed. Evaluation is done lazily.
 
+Params:
+    fun = one or more functions
+    r = an input range
+
+Returns:
+    r with each fun applied to all the elements
```

Comment:
This does not make it clear what exactly is returned when more than one function is specified in `fun`. For example, what does `map!("a+a", "a*a")` return?